# Daily
## Nanopore
### é‡å¤åºåˆ—åˆ†æ

å¾®å«æ˜Ÿæ ‡è®°ï¼ˆmicrosatelliteï¼‰ï¼Œåˆè¢«ç§°ä¸ºçŸ­ä¸²è”é‡å¤åºåˆ—ï¼ˆshort tandem repeats, STRsï¼‰æˆ–ç®€å•é‡å¤åºåˆ—ï¼ˆsimple sequence repeats, SSRsï¼‰ï¼Œæ˜¯å‡åŒ€åˆ†å¸ƒäºçœŸæ ¸ç”Ÿç‰©åŸºå› ç»„ä¸­çš„ç®€å•é‡å¤åºåˆ—ï¼Œç”±2ï½6ä¸ªæ ¸è‹·é…¸çš„ä¸²è”é‡å¤ç‰‡æ®µï¼ˆæ ¸å¿ƒåºåˆ—ï¼‰ä¸²è”é‡å¤ç»„æˆï¼Œå…¶é‡å¤å•ä½çš„é‡å¤æ¬¡æ•°åœ¨ä¸ªä½“é—´å‘ˆé«˜åº¦å˜å¼‚æ€§å¹¶ä¸”æ•°é‡ä¸°å¯Œã€‚ç›®å‰å·²å‘ç°é‡å¤åºåˆ—å’Œ40å¤šç§ç¥ç»è‚Œè‚‰å’Œç¥ç»é€€è¡Œæ€§ç–¾ç—…ç­‰ç–¾ç—…æœ‰å…³ï¼ŒåŒ…æ‹¬ä¼—æ‰€ç†ŸçŸ¥çš„ç²¾ç¥å‘è‚²è¿Ÿæ»ç–¾ç—…â€”è„†æ€§XæŸ“è‰²ä½“ã€ç¥ç»é€€è¡Œæ€§ç–¾ç—…â€”äº¨å»·é¡¿èˆè¹ˆç—‡ã€è„Šé«“å°è„‘æ€§å…±æµå¤±è°ƒç—‡ç­‰ï¼Œæ­¤å¤–å¾®å«æ˜Ÿä¸ç¨³å®šæ€§MSIä¹Ÿæ˜¯è®¸å¤šç™Œç—‡åŸºå› ç»„ç‰¹å¾ã€‚ç”±äºé‡å¤åºåˆ—çš„æ‰©å¼ å¼•èµ·çš„ç–¾ç—…ç§°ä¸ºé‡å¤åºåˆ—æ‰©å¼ ç–¾ç—…ï¼Œå½“ç„¶æœ‰äº›é‡å¤åºåˆ—ç¼©çŸ­ä¹Ÿèƒ½å¼•èµ·ç–¾ç—…ã€‚å…¶å‘ç—…æœºåˆ¶ä¸è¿™äº›é‡å¤çš„å¾®å«æ˜Ÿåºåˆ—çš„é‡å¤æ¬¡æ•°ç›¸å…³ã€‚åˆ©ç”¨ä¸‰ä»£é•¿è¯»é•¿æ•°æ®ï¼Œå¯ç”¨æ¥æ£€æµ‹å«æ˜Ÿåºåˆ—é‡å¤æ¬¡æ•°ã€‚

### HLAåˆ†å‹

ä¸»è¦ç»„ç»‡ç›¸å®¹æ€§å¤åˆä½“MHCåŒºåŸŸä½äº6å·æŸ“è‰²ä½“çš„çŸ­è‡‚ä¸Šï¼Œæ˜¯äººç±»åŸºå› ç»„ä¸Šæœ€å¤æ‚çš„åŒºåŸŸä¹‹ä¸€ï¼ˆçº¦4Mbï¼‰ï¼Œå‘ˆç°å‡ºé«˜åº¦çš„å¤šæ€æ€§ï¼ˆæœ‰ç€è¶…è¿‡10,000ä¸ªç­‰ä½ï¼‰ã€‚å…¶ç¼–ç çš„åˆ†å­å‚ä¸æŠ—åŸé€’å‘ˆï¼Œåˆ¶çº¦ç»†èƒé—´ç›¸äº’è¯†åˆ«åŠè¯±å¯¼å…ç–«åº”ç­”ã€‚äººç±»ç™½ç»†èƒæŠ—åŸï¼ˆHLAï¼‰ç¼–ç åŸºå› æ˜¯ MHC çš„ä¸€éƒ¨åˆ†ï¼Œæ˜¯è¿„ä»Šå·²çŸ¥åŸºå› ä¸­ç­‰ä½åŸºå› å¤šæ€æ€§æœ€é«˜çš„åŸºå› å¤åˆä½“ï¼Œä¹Ÿæ˜¯ä¸åŒä¸ªä½“è¿›è¡Œå™¨å®˜æˆ–ç»„ç»‡ç»†èƒç§»æ¤æ—¶å‘ç”Ÿæ’æ–¥çš„ä¸»è¦æˆåˆ†ã€‚

ä¸ HLA ç›¸å…³çš„ç–¾ç—…å¤šè¾¾100å¤šç§ï¼Œæ¶‰åŠè‡ªèº«å…ç–«æ€§ç–¾ç—…ã€å…ç–«ç¼ºé™·æ€§ç–¾ç—…ã€è¿‡æ•æ€§ç–¾ç—…ã€æ„ŸæŸ“ç±»ç–¾ç—…ã€ä»£è°¢æ€§ç–¾ç—…ç­‰ï¼Œå¦‚ç³–å°¿ç—…ã€ç±»é£æ¹¿æ€§å…³èŠ‚ç‚ï¼Œé“¶å±‘ç—…ã€å¼ºç›´æ€§è„ŠæŸ±ç‚ã€é‡ç—‡è‚Œæ— åŠ›å’Œå“®å–˜ç­‰ã€‚åŒæ—¶ï¼ŒHLAåœ¨å™¨å®˜å’Œéª¨é«“ç­‰ç§»æ¤ä¸­èµ·åˆ°è‡³å…³é‡è¦çš„ä½œç”¨ï¼Œä¹Ÿä¸è®¸å¤šè¯ç‰©çš„ä¸¥é‡ä¸è‰¯ååº”ç›¸å…³ã€‚å› æ­¤ï¼Œè¿›è¡ŒHLA åˆ†å‹ï¼Œæœ‰åˆ©äºå…ç–«ç›¸å…³ç–¾ç—…çš„ç ”ç©¶ã€ç–«è‹—å’Œè¯ç‰©é¶å‘äººç¾¤ç­›é€‰ã€ç§æ—è¿›åŒ–çš„ç ”ç©¶ã€ç»„ç»‡å’Œå™¨å®˜ç§»æ¤ç­‰ã€‚

æœ¬æ¬¡åˆ†æå¯¹æ ·å“çš„ HLA-Aï¼ŒHLA-Bï¼ŒHLA-C åŸºå› è¿›è¡Œå•å€å‹é‰´å®šã€‚å°†æµ‹åºçš„NanoporeÂ readsä¸å·²çŸ¥çš„HLAç­‰ä½æ¯”å¯¹æ¥è¯†åˆ«å€™é€‰çš„ç­‰ä½ï¼Œæ¥ä¸‹æ¥é€šè¿‡ä¸å€™é€‰ç­‰ä½çš„å¤šé‡æ¯”å¯¹è·å–ä¸€è‡´åºåˆ—ï¼Œæœ€åé€šè¿‡å°†ä¸€è‡´åºåˆ—ä¸å‚è€ƒæ•°æ®åº“æ¯”å¯¹è·å–æ¯ä¸ªæ ·å“æœ€ç»ˆçš„å•å€å‹ã€‚

## æµ‹è¯•call snpè½¯ä»¶
### freebayes
```
freebayes -f /datapool/home/wangmc/project/genome/ucsc.hg19.fasta /datapool/home/wangmc/project/case_22BY12800/case_22BY12800.bam > case_22BY12800.vcf
```
freebayeså‚æ•°æ„ä¹‰<br>
freebayesç»“æœè¿‡æ»¤æ ‡å‡†<br>

### deepvariant
#### 1 condaå®‰è£…ä¸å¯è¡Œ pass
#### 2 ä¸‹è½½æºç ç¼–è¯‘å®‰è£…ï¼Œä¾èµ–å¤ªå¤šï¼Œéœ€è¦ç®¡ç†å‘˜æƒé™ä¹Ÿä¸å¯è¡Œ pass
#### 3 singularityå¯ä»¥æˆåŠŸå®‰è£…å¹¶ä½¿ç”¨
1 æ‹‰é•œåƒï¼š<br>
```
singularity build deepvariant.sif docker://google/deepvariant

singularity pull deepvariant.sif docker://google/deepvariant:latest
```
2 è¿è¡Œï¼š<br>
```
singularity run -B /usr/lib/locale/:/usr/lib/locale/  /datapool/home/wangmc/software/deepvariant/deepvariant.sif /opt/deepvariant/bin/run_deepvariant --model_type=WES --ref=/datapool/home/wangmc/project/genome/ucsc.hg19.fasta --reads=/datapool/home/wangmc/project/case_22BY12800/case_22BY12800.bam  --output_vcf=case_22BY12800.vcf --output_gvcf=case_22BY12800.g.vcf case_22BY12800 --num_shards=30
```
bamå’Œå‚è€ƒåŸºå› ç»„fastaç§»åŠ¨åˆ°ä¸€ä¸ªç£ç›˜ä¸‹,dockerè°ƒç”¨ç³»ç»Ÿæ–‡ä»¶çš„é—®é¢˜<br>
åŒæ—¶ç§»åˆ°/homeç›®å½•ä¸‹
[SAMtools [E::hts_open_format] Failed to open file #1289](https://github.com/samtools/samtools/issues/1289)

### bcftools
```
/datapool/bioinfo/xinghu/Biotools/miniconda/bin/bcftools mpileup -Ou -R case_22BY12800.bed --threads 30 -f /datapool/home/wangmc/project/genome/ucsc.hg19.fasta /datapool/home/wangmc/project/case_22BY12800/case_22BY12800.bam | /datapool/bioinfo/xinghu/Biotools/miniconda/bin/bcftools call --threads 30 -mv -Ov -o case_22BY12800.vcf
```

### ä¸åŒcall snpè½¯ä»¶ç»“æœè¯„ä¼°æ ‡å‡†
#### 1 snpäº¤é›†


### DeepNull GWAS
https://github.com/google-health/genomics-research/tree/main/nonlinear-covariate-gwas<br>
å»ºç«‹éçº¿æ€§åå˜é‡æ•ˆåº”æ¨¡å‹ï¼Œæé«˜è¡¨å‹é¢„æµ‹å’Œå…³è”èƒ½åŠ›<br>

### gVCFå’ŒVCFçš„åŒºåˆ«
https://github.com/broadgsa/gatk/blob/master/doc_archive/faqs/What_is_a_GVCF_and_how_is_it_different_from_a_'regular'_VCF%3F.md<br>
https://support.illumina.com/help/BaseSpace_OLH_009008/Content/Source/Informatics/BS/gVCFFiles_Intro_swBS.htm<br>
gvcfæ–‡ä»¶ä¼šè®°å½•æ›´å¤šçš„ä¿¡æ¯ï¼Œè¿™é‡Œæ›´å¤šçš„ä¿¡æ¯æŒ‡çš„æ˜¯æœªçªå˜çš„ä½ç‚¹çš„è¦†ç›–æƒ…å†µï¼Œgvcfæ–‡ä»¶ä¹Ÿåˆ†ä¸¤ç§ï¼Œä¸€ç§æ˜¯-erc gvcf ï¼Œå¦ä¸€ç§æ˜¯ -erc bp_resolution,è¿™ä¸¤ç§gvcfæ–‡ä»¶çš„åŒºåˆ«åœ¨äºå‰ä¸€ç§gvcfæ–‡ä»¶è®°å½•éçªå˜ä½ç‚¹çš„æ—¶å€™ï¼Œä»¥å—çš„å½¢å¼æ¥è®°å½•ï¼Œè€Œåä¸€ç§gvcfæ–‡ä»¶åˆ™æ˜¯å¯¹éçªå˜å’Œçªå˜ä½ç‚¹ä¸€è§†åŒä»ï¼Œå‰ä¸€ç§æ–¹å¼æ˜¯ä¸ºäº†æœ‰æ•ˆçš„å‹ç¼©æ–‡ä»¶çš„è¡Œæ•°å’Œå¤§å°ï¼Œå¯¹åç»­çš„åˆ†ææ²¡æœ‰å½±å“ï¼Œå› æ­¤è¿™é‡Œæ¨èä½¿ç”¨å‰ä¸€ç§gvcfæ–‡ä»¶ã€‚
é‚£ä¹ˆä¸ºä»€ä¹ˆè¦ä½¿ç”¨gvcfæ–‡ä»¶è€Œä¸æ˜¯vcfæ–‡ä»¶å‘¢ï¼Ÿè¿™é‡Œä¸»è¦çš„åŸå› åœ¨äºå¤šä¸ªæ ·æœ¬çš„vcfæ–‡ä»¶è¿›è¡Œåˆå¹¶çš„æ—¶å€™ï¼Œéœ€è¦åŒºåˆ†./.å’Œ0/0çš„æƒ…å†µï¼Œ./.æ˜¯æœªæ£€å‡ºçš„åŸºå› å‹ï¼Œè€Œ0/0æ˜¯æœªçªå˜çš„åŸºå› å‹ï¼Œå¦‚æœä»…ä½¿ç”¨æ™®é€šçš„vcfæ–‡ä»¶è¿›è¡Œåˆå¹¶ï¼Œé‚£ä¹ˆå°±æ— æ³•åŒºåˆ†è¿™ä¸¤ç§æƒ…å†µï¼Œè¿›è€Œå¯¹åˆå¹¶ç»“æœäº§ç”Ÿåå·®ã€‚å®é™…ä¸Šï¼Œæˆ‘ä»¬ä¹Ÿå¯ä»¥ç›´æ¥å°†gvcfæ–‡ä»¶å’Œvcfæ–‡ä»¶ä½¿ç”¨bcftools mergeè¿›è¡Œmergeï¼Œä½†æ˜¯è¿™æ ·æ‹¿åˆ°çš„ç»“æœä¼šæœ‰åå·®ï¼Œå› ä¸ºvcfæ–‡ä»¶æ²¡æœ‰æœªçªå˜çš„ä½ç‚¹çš„æƒ…å†µã€‚

### illuminaå®˜ç½‘æ”¯æŒ

githubä¸»é¡µå€¼å¾—å…³æ³¨ï¼šhttps://github.com/Illumina<br>

https://support.illumina.com/content/dam/illumina-support/help/Illumina_DRAGEN_Bio_IT_Platform_v3_7_1000000141465/Content/SW/Informatics/Dragen/QUAL_QD_GQ_Formulation_fDG.htm

### ä¸ºä»€ä¹ˆVCFä¸­Allele Depthï¼ˆADï¼‰æ¯”é¢„æœŸçš„ä½å¥½å¤š,ç†è®ºä¸ŠADä¹‹å’Œåº”è¯¥ç­‰äºDP
https://www.jianshu.com/p/401202306f41<br>
https://gatk.broadinstitute.org/hc/en-us/articles/360035532252?id=11096<br>

### bam-readcount bamæ–‡ä»¶ç»Ÿè®¡
https://github.com/genome/bam-readcount<br>
bam-readcount is a utility that runs on a BAM or CRAM file and generates low-level information about sequencing data at specific nucleotide positions. Its outputs include observed bases, readcounts, summarized mapping and base qualities, strandedness information, mismatch counts, and position within the reads. 
### kmdiff
kmdiff provides differential k-mers analysis between two populations (control and case), Each population is represented by a set of short-read sequencing. Outputs are differentially represented k-mers between controls and cases.
### å…ç–«ç»„åº“åˆ†æ
https://github.com/milaboratory/mixcr<br>


## æœºå™¨å­¦ä¹ åœ¨ç”Ÿç‰©ä¿¡æ¯ä¸­çš„åº”ç”¨

### æ·±åº¦å­¦ä¹ 
#### å…¨è¿æ¥ç½‘ç»œ
#### æ·±åº¦å·ç§¯
#### å¾ªç¯å·æœº
#### å›¾å·æœº

### ğŸŒ°
#### Clair3ï¼šä¸‰ä»£æµ‹åºreadså˜å¼‚æ£€æµ‹
https://github.com/HKU-BAL/Clair3<br>

## condaé•œåƒæºé…ç½®
```
conda config --add channels https://mirrors.aliyun.com/anaconda/pkgs/main/
conda config --add channels https://mirrors.aliyun.com/anaconda/cloud/conda-forge/
conda config --add channels https://mirrors.aliyun.com/anaconda/cloud/bioconda/
```
## ä¸‰ä»£æµ‹åºæ•°æ®åˆ†æå·¥å…·
https://long-read-tools.org/index.html<br>

## condaæ›´æ–°æˆ–pipæ›´æ–°å¯¼è‡´å·²å®‰è£…åŒ…é”™è¯¯ï¼Œéœ€è¦é‡æ–°å®‰è£…

```
Obtaining file:///datapool/home/wangmc/software/cnvkit
  Preparing metadata (setup.py) ... error
  error: subprocess-exited-with-error
  
è§£å†³æ–¹æ¡ˆé‡æ–°å®‰è£…importlib_metadata
pip uninstall importlib_metadata
pip install importlib_metadata --force-reinstall

```
## googelé•œåƒç½‘é¡µ
https://yimili.net/google-seach/<br>
https://www.library.ac.cn/<br>

## ä¸‹è½½sciæ–‡çŒ®
https://kgithub.com/Tishacy/SciDownl

## gwasæŸ¥è¯¢
https://github.com/ramiromagno/gwasrapidd/

## æ ¼å¼è½¬æ¢
https://kgithub.com/bioconvert/bioconvert

## goè¯­è¨€é‡å†™æ¨¡å—åŠ é€Ÿgatk callå˜å¼‚æµç¨‹
https://github.com/ExaScience/elprep

## æ€è°‹å­¦æœ¯
https://ac.scmor.com/

## pythonç¼–è¯‘ï¼ŒåŠ é€Ÿç¨‹åºè¿è¡Œ
https://github.com/exaloop/codon

## Python readline()å’Œreadlines()å‡½æ•°

## CNVè½¯ä»¶æµ‹è¯„
https://github.com/StahlKt/ImputationComparisonPaper2021

## å•ç»†èƒæ–‡çŒ®åˆ†æä»£ç 
https://github.com/Xiaxy-XuLab/PanCAF

## goè¯­è¨€æ·»åŠ å›½å†…é•œåƒæºå¤´
```
export GO111MODULE=on
export GOPROXY=https://goproxy.cn
 ```
## PRSåˆ†æ
```
https://github.com/choishingwan/PRSice
```
## äººç±»T2TåŸºå› ç»„
https://github.com/marbl/CHM13

## bamæ–‡ä»¶æ·±åº¦ç»Ÿè®¡ç­‰ï¼Œç›¸å…³bedtools
https://kgithub.com/brentp/mosdepth

## vcfstats, ç»Ÿè®¡vcfå„é¡¹æŒ‡æ ‡ï¼Œæ•°é‡ã€ç­‰ä½åŸºå› é¢‘ç‡ï¼›
https://kgithub.com/pwwang/vcfstats

## fastaã€sameã€bedã€vcfæ ¼å¼ç”¨pythoné‡å†™çš„åŒ…ï¼Œæ–¹ä¾¿è°ƒç”¨
https://github.com/daler/pybedtools<br>
https://github.com/mdshw5/pyfaidx<br>
https://github.com/brentp/cyvcf2<br>
https://github.com/pysam-developers/pysam<br>
https://bedops.readthedocs.io/en/latest/index.html<br>

## bayes
http://allendowney.github.io/ThinkBayes2/index.html

## python argsort()å‡½æ•°
 ```
 argsort()å‡½æ•°æ˜¯å°†xä¸­çš„å…ƒç´ ä»å°åˆ°å¤§æ’åˆ—ï¼Œæå–å…¶å¯¹åº”çš„index(ç´¢å¼•å·)
 ```

## å“ˆä½›è½¯ä»¶
https://www.hsph.harvard.edu/alkes-price/software/

## strobealignç±»ä¼¼bwaçš„æ¯”å¯¹å·¥å…·
https://github.com/ksahlin/strobealign

## ç”Ÿä¿¡ç»éªŒ
https://eriqande.github.io/eca-bioinf-handbook/

## lncRNAåˆ†æå·¥å…·æµ‹è¯„
https://github.com/cbl-nabi/RNAChallenge

## äººç±»å…¨åŸºå› ç»„Ti/Tv
https://bioinformatics.stackexchange.com/questions/4362/why-ti-tv-ratio#:~:text=It%20is%202%E2%80%932.10%20for%20human%20across%20the%20whole,Also%2C%20this%20number%20is%20correlated%20with%20GC%20content

## ç”»å›¾
https://github.com/IndrajeetPatil/ggstatsplot/

## æ„ŸäººåŠ±å¿—çš„çœŸäººçœŸäº‹
http://www.cureffi.org/about/

## 125ä¸ªå‰æ²¿ç§‘å­¦é—®é¢˜
https://www.science.org/content/resource/125-questions-exploration-and-discovery

## chip-seq atac-seq hic-seq æ¯”å¯¹å·¥å…·
https://github.com/haowenz/chromap

## Hi-C scaffoldingå·¥å…·
https://github.com/c-zhou/yahs

## VCFæ–‡ä»¶åˆ†æPCAå’Œé—ä¼ è·ç¦»
https://github.com/zhengxwen/SNPRelate

## githubè®¿é—®åŠ é€Ÿ
https://github.com/521xueweihan/GitHub520

## strand-flipping
Flipping strand means changing alleles A -> T C -> G G -> C T -> A s

## è¿é”åˆ†æè®¡ç®—LOD
https://www.wikihow.com/Calculate-LOD-Score<br>
https://www.mun.ca/biology/scarr/LOD_analysis.html<br>
https://med.libretexts.org/Bookshelves/Basic_Science/Cell_Biology_Genetics_and_Biochemistry_for_Pre-Clinical_Students/14%3A_Linkage_studies_pedigrees_and_population_genetics/14.03%3A_Linkage_analysis_and_Genome_Wide_Association_Studies_(GWAS)<br>

## 1000 Genomics è¿é”ä¸å¹³è¡¡
https://github.com/CBIIT/LDlinkR<br>

## å¤šåºåˆ—æ¯”å¯¹(multiple sequence align)
kalign: https://github.com/TimoLassmann/kalign<br>
maftt: https://mafft.cbrc.jp/alignment/software/linux.html<br>
muscel: https://github.com/rcedgar/muscle<br>
multiz: https://github.com/multiz/multiz<br>

## æ³›åŸºå› ç»„
https://github.com/pangenome<br>
https://github.com/vgteam<br>

## cDNA/EST proteinæ¯”å¯¹åˆ°åŸºå› ç»„
https://github.com/ogotoh/spaln<br>

## ç»“æ„å˜å¼‚æµç¨‹ï¼ˆå¤šè½¯ä»¶æ•´åˆï¼‰
https://github.com/stat-lab/MOPline

## HLAåŸºå› å‘½åè§„åˆ™å’Œåˆ†å‹æ•°æ®åº“
å‘½åï¼š https://hla.alleles.org/nomenclature/naming.html
åˆ†å‹ï¼š https://www.ebi.ac.uk/ipd/imgt/hla/

## mac brew å®‰è£…æ›´æ–°æ…¢ï¼Œè¯•è¯•å›½å†…é•œåƒæº
https://mirrors.tuna.tsinghua.edu.cn/help/homebrew/

## æ·±åº¦å­¦ä¹ ä»å¤´é¢„æµ‹åŸºå› 
https://github.com/weberlab-hhu/helixer

## ç‹ä¸€å“ˆå¸Œï¼Œè™½ç„¶ä¸æ˜¯å¾ˆæ‡‚ï¼Œç¡®å®åšäº†å¾ˆæœ‰æ„ä¹‰çš„ä¸€ä»¶äº‹
https://github.com/wangyi-fudan/wyhash

## å¤§ç‰›çš„åšå®¢ï¼Œç®€å•åˆæœ´ç´ 
http://norvig.com/

## è®©ä½ çš„pythonç¨‹åºæ˜¾ç¤ºè¿›åº¦æ¡
https://github.com/tqdm/tqdm

## ç®—æ³•ï¼ˆAlgorithms, 4th Editionï¼‰
https://algs4.cs.princeton.edu/home/

## CS è‡ªå­¦æŒ‡å—ï¼ˆè¦æ˜¯å¤§ä¸€çš„æ—¶å€™çœ‹åˆ°å°±å¥½äº†ï¼‰
https://github.com/PKUFlyingPig/cs-self-learning

## ç”¨äºæµ‹è¯„callå˜å¼‚çš„å‡†ç¡®æ€§ï¼Œå¯ä½œä¸ºGenome in a Bottle (GIAB)çš„è¡¥å……
https://genomebiology.biomedcentral.com/articles/10.1186/s13059-023-03109-2

## å¤§äºŒæˆ–å¤§ä¸‰å¯¹ç¼–ç¨‹è¯­è¨€å°±æœ‰å¾ˆé«˜çš„é€ è¯£ï¼Œå‡ä»¥æ—¶æ—¥å¿…æˆæ ‹æ¢ï¼Œç¥å¥½
https://monthly.wybxc.cc/

## ä¸åŒæ‚åˆåº¦åŸºå› ç»„çš„ç»„è£…æŒ‡å—
https://doi.org/10.1093/bib/bbad337

##  cå†™çš„å¾ˆå¥½
åœ¨1ä¸ªå¾ˆå°å¾ˆå°çš„é¢†åŸŸé‡Œåšåˆ°æè‡´ï¼Œè¿™äº›é™Œç”Ÿäººçš„ä»£ç ï¼ŒåŠ ä¹‹æœ¬èº«å¯¹å…¶äººç”Ÿçš„æƒ³è±¡ï¼Œæ‰€è°“åŠŸæˆåå°±èµšå¤§é’±ä¸æ˜¯å¿…é¡»çš„ï¼Œæœ€èƒ½æ‰“åŠ¨æˆ‘çš„è«è¿‡äºå¯¹æŸæ ·ä¸œè¥¿æŒä¹‹ä»¥æ’çš„é’»ç ”å’Œçƒ­çˆ±ï¼Œç„¶åå¯¹çŸ¥è¯†çš„æ— ç§åˆ†äº«ï¼Œå› ä¸ºè¿™æ ·çš„äººå¯¹è¿™ä¸ªä¸–ç•Œå§‹ç»ˆæ€€æœ‰ç¾å¥½çš„æœŸå¾…å§ï¼Œæ€»æ˜¯åœ¨ç²¾ç¥ä¸Šç»™äºˆæˆ‘å¾ˆå¤§çš„æ…°è—‰ï¼Œè§‰å¾—è‡ªå·±ä¹Ÿå¯ä»¥æˆä¸ºä¸€ä¸ªé»˜é»˜æ— é—»è®©ä¸–ç•Œå˜å¥½çš„æ™®é€šäºº<br>
https://attractivechaos.wordpress.com/

## IBD
https://github.com/browning-lab/hap-ibd?tab=readme-ov-file<br>

## PrimateAI-3D ç”¨äºé—ä¼ å˜å¼‚è§£è¯»ï¼Œå¯ä»¥å’ŒAlphaMissenseæ¯”è¾ƒè¡¥å……
https://primad.basespace.illumina.com/

## æ ‡å‡†è¯¯å’Œæ ‡å‡†å·®çš„æ¯”è¾ƒ
http://www.differencebetween.net/science/mathematics-statistics/difference-between-standard-deviation-and-standard-error<br>

## bcftoolsæ’ä»¶ï¼Œhg38,hg19,chm13åŸºå› ç»„åæ ‡ç›¸äº’è½¬æ¢ã€gwas metaåˆ†æã€å¤šåŸºå› è¯„åˆ†ã€BLUPé¢„æµ‹ç–¾ç—…çš„é£é™©ã€PLINK/PLINK2/REGENIE/SAIGE/BOLT/METAL/PGS/SSFä¿¡æ¯æ±‡æ€»ã€ polygenic score loadings tool
https://github.com/freeseek/score<br>

## ç²¾ç¥ç±»ç–¾ç—…gwas summaryç»“æœæ±‡é›†
https://pgc.unc.edu/for-researchers/download-results/<br>



 
 
